source/manual/fall-back-to-aws-cloudfront.html.md

```diff
@@ -23,49 +23,78 @@ This backup CDN is currently provided by AWS CloudFront.
+
+This [Draft PR to Failover to AWS CloudFront](https://github.com/alphagov/govuk-dns-tf/pull/69) shows the `CNAME`s you need to change, and how to test that they are correct
+
+You can also get the `CNAME`s to use for the secondary CDN from the AWS CLI:
+
+```bash
+  # www-cdn.production.govuk.service.gov.uk
+  gds aws govuk-production-readonly aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Aliases.Items[0]=='www.gov.uk'].DomainName | [0]"
+  # assets.publishing.service.gov.uk
+  gds aws govuk-production-readonly aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Aliases.Items[0]=='assets.publishing.service.gov.uk'].DomainName | [0]"
+```
+
+### Staging
+
+For **staging DNS records**, open the following three pages as separate tabs:
+
+- [GCP Cloud DNS www.staging.publishing.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/publishing-service-gov-uk/rrsets/www.staging.publishing.service.gov.uk./CNAME/edit-standard?project=govuk-production)
+- [GCP Cloud DNS assets.staging.publishing.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/publishing-service-gov-uk/rrsets/assets.staging.publishing.service.gov.uk./CNAME/edit-standard?project=govuk-production)
+- [AWS Route 53 publishing.service.gov.uk](https://us-east-1.console.aws.amazon.com/route53/v2/hostedzones#ListRecordSets/Z3SBFBO09PD5HF)
+
+You are going to update the `CNAME` records for two different domains, in both GCP and AWS:
+
+For **staging**, these two domains are `www.staging.publishing.service.gov.uk` and `assets.staging.publishing.service.gov.uk`
+You can get the `CNAME`s to use for the secondary CDN from the AWS CLI:
+
+```bash
+  # www.staging.publishing.service.gov.uk
+  gds aws govuk-staging-readonly aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Aliases.Items[0]=='www.staging.publishing.service.gov.uk'].DomainName | [0]"
+  # assets.staging.publishing.service.gov.uk
+  gds aws govuk-staging-readonly aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Aliases.Items[0]=='assets.staging.publishing.service.gov.uk'].DomainName | [0]"
+```
+
+### Next steps
+
+In either case, the records should look like `d0000000000000.cloudfront.net.` (with 0s replaced with letters and numbers). Now you can manually update the `CNAME` records for both domains in both GCP and AWS, via the tabs you opened in your web browser earlier:
+
+- Change the canonical name from `www-gov-uk.map.fastly.net.` to the CloudFront domain name you found before, including the trailing period (e.g. `d0000000000000.cloudfront.net.`)
+- Test if the new Cloudfront domain is serving assets correctly, for example:
+
+```bash
+curl -vs https://assets.staging.publishing.service.gov.uk/media/662a74aa45f183ec818a72c2/dvsa-earned-recognition-vehicle-operators-accredited-list.csv/preview | grep cloudfront
```

Comment:
I think we need to tell the dev what to look for from the curl response that will inform them that the requests are being served by the updated CDN - 

missing key - `fastly-backend-name`

or the presence of `xxxxx.cloudfront.net` value in the `via` key.

Reply:
Updated with your suggestions